common/src/models/error.rs

```diff
@@ -91,14 +91,14 @@ impl From<ErrorKind> for ApiError {
                 (StatusCode::FORBIDDEN, "You cannot create more projects. Delete some projects first.")
             },
             ErrorKind::ProjectHasRunningDeployment => (
-                StatusCode::FORBIDDEN,
-                "A deployment is running. Stop it with `cargo shuttle stop` first."
+                StatusCode::INTERNAL_SERVER_ERROR,
+                "Could not automatically stop the running deployment for the project. Reach out to our support and ask them for a helping hand."
```

Comment:
Nit: should we rephrase the last sentence to: `Please reach out to Shuttle support for help.`. Same for the ProjectHasResources one.